src/components/Chart.tsx

```diff
@@ -0,0 +1,156 @@
+/** Load dependencies */
+import React from "react";
+import { Chart as ChartJS } from "chart.js";
+import "chartjs-plugin-datalabels";
+
+/** Load types */
+import { Metric } from "../types";
+
+/** Types */
+type Context =
+  | string
+  | CanvasRenderingContext2D
+  | HTMLCanvasElement
+  | ArrayLike<CanvasRenderingContext2D | HTMLCanvasElement>;
+
+type RefContext =
+  | string
+  | ((instance: HTMLCanvasElement | null) => void)
+  | React.RefObject<HTMLCanvasElement>
+  | null
+  | undefined;
+
+export type ChartTypes =
+  | "line"
+  | "bar"
+  | "radar"
+  | "pie"
+  | "doughnut"
+  | "polarArea";
+
+/** Interfaces */
+interface Datalabels {
+  anchor: string;
+  align: string;
+  formatter: (x: number) => number;
+  font: {
+    weight: string;
+  };
+}
+
+interface ChartProps {
+  className?: string;
+  data: {
+    labels: Metric["label"][];
+    values: Metric["value"][];
+  };
+  types?: ChartTypes[];
+}
+
+/** Component */
+const Chart: React.FunctionComponent<ChartProps> = ({
+  className,
+  data,
+  types,
+}: ChartProps) => {
```

Comment:
```suggestion
}) => {
```
That's just a _tip_ for you, you don't have to explicit declare the type here, this `React.FunctionComponent<ChartProps>` already has the signature of the whole method :)